__tests__pet.test.js

```diff
@@ -0,0 +1,99 @@
+const Pet = require('./src/pet');
+
+describe ('constructor', () => {
+    it('returns an object', () => {
```

Comment:
As we're creating a new instance of `Pet` for each test here, it would make sense to use a `beforeEach` function to do this for us and save the repetition:
```
    let pet;
    beforeEach(() => {
      pet = new Pet("Fido");
    });
```
Doing this would mean you can remove all the `const pet = new Pet('Fido')` declarations in each test. In the case of the last test, you could just change `mamaBear` to `pet` and it would still work fine